README.md

```diff
@@ -8,5 +8,5 @@ asldfjalsdfjhlaksjdfhlaskjdf
 ### example
 
 ```cpp 
-int x = 0;aslkdjf;alskdjf;aklsjdf
```

Comment:
this is not something i can put into main
there is a bug here and the code is not clear enough, what does it do?